etc/picongpu/davide-cineca/gpu_picongpu.profile.example

```diff
@@ -0,0 +1,101 @@
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj=$(groups | awk '{print $2}')
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load gnu/6.4.0
+module load cmake/3.11
+module load cuda/9.2.88
+module load openmpi/3.1.0--gnu--6.4.0
+module load boost/1.68.0--openmpi--3.1.0--gnu--6.4.0
+
+export CMAKE_PREFIX_PATH=$CUDA_HOME:$OPENMPI_HOME:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=$BOOST_HOME:$CMAKE_PREFIX_PATH
+
+# Other Software ##############################################################
+#
+module load zlib/1.2.11--gnu--6.4.0
+module load szip/2.1.1--gnu--6.4.0
+module load blosc/1.12.1--gnu--6.4.0
+
+module load hdf5/1.10.4--openmpi--3.1.0--gnu--6.4.0
+module load libsplash/1.7.0--openmpi--3.1.0--gnu--6.4.0
+module load adios/1.13.1--openmpi--3.1.0--gnu--6.4.0
+
+module load libpng/1.6.35--gnu--6.4.0
+module load freetype/2.9.1--gnu--6.4.0
+module load pngwriter/0.7.0--gnu--6.4.0
+
+export CMAKE_PREFIX_PATH=$ZLIB_HOME:$SZIP_HOME:$BLOSC_HOME:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=$HDF5_HOME:$LIBSPLASH_HOME:$ADIOS_HOME:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=$LIBPNG_HOME:$FREETYPE_HOME:$PNGWRITER_HOME:$CMAKE_PREFIX_PATH
+
+# Work-Arounds ################################################################
+#
+# fix for Nvidia NVCC bug id 2448610
+# see https://github.com/ComputationalRadiationPhysics/alpaka/issues/701
+export CXXFLAGS="-Dlinux"
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
+
+export PICSRC=$HOME/src/picongpu
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:60"
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+#   - SLURM (sbatch)
+#   - "gpu" queue
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE="etc/picongpu/davide-cineca/gpu.tpl"
+
+# allocate an interactive shell for one hour
+#   getNode 2  # allocates to interactive nodes (default: 1)
+function getNode() {
+    if [ -z "$1" ] ; then
+        numNodes=1
+    else
+        numNodes=$1
+    fi
+    srun --time=0:30:00 --nodes=$numNodes --ntasks-per-socket=8 --ntasks-per-node=16 --mem=254000 --gres=gpu:4 -A $proj -p dvd_usr_prod --pty bash
```

Comment:
After fighting since hours with SLURM I think we should allocate a node on the following way:

```
export OPEN_MPI_NUM_THREADS=16
salloc --time=0:30:00  --nodes=$numNodes --ntasks=$(( $numNodes * 4)) --ntasks-per-socket=2  --time=00:30:00 --gres=gpu:4 --gres-flags=enforce-binding --cpus-per-task=16 -A $proj -p dvd_usr_prod --mem=254000
```

IN the command above I changed `srun` to `salloc` because this will give the user the possibility to call PIConGPU with `srun picongpu ...`. To use `srun` within an interactive shell allocated with `srun ... --pty bash` is not possible (will hang). The only way, in that case, would be to use mpiexec which than not fits the user guides of the facility.
`--cpus-per-task=16` will give 16 cores to each tasks. Normally it must be 32 but this is not accepted from the batch system.

Reply:
The disadvantage of using salloc is that you must always use `srun` to execute a command on the compute node.

Reply:
> Are you sure we need to support interactive multi-node allocations? Usually one full interactive node with its multiple GPUs will just do.

This is independent of single or multi-node. The problem is that it is not possible to start any MPI program with srun if we are already in a srun environment.

Which method (srun or salloc) is the best solution depends on the workflow. I use  the `getNode` command very often to run PIConGPU (via. `./picongpu`) but if `tbg -t bash ...` is used or the node should be used to compile PIConGPU than the current way is right.